app/src/main/java/org/wikipedia/page/references/References.java

```diff
@@ -0,0 +1,64 @@
+package org.wikipedia.page.references;
+
+import androidx.annotation.NonNull;
+
+import com.google.gson.annotations.SerializedName;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.List;
+import java.util.Map;
+
+public class References {
```

Comment:
If you put the `@SuppressWarnings("unused,NullableProblems")` annotation at the top of this class itself, you won't need to put it for every field.

Reply:
Thanks! You're correct!